test/parallel/test-crypto-sign-verify.js

```diff
@@ -15,6 +15,40 @@ const certPem = fixtures.readSync('test_cert.pem', 'ascii');
 const keyPem = fixtures.readSync('test_key.pem', 'ascii');
 const modSize = 1024;
 
+{
+  const Sign = crypto.Sign;
+  const instance = Sign('SHA256');
+  assert.ok(instance instanceof Sign, 'call sign constructor without new');
```

Comment:
The message here seems to be more of a description of the test than communicate what was expected. Maybe something like "expected Sign to return a new instance when called without `new` keyword"?

Reply:
Thank you for review.
I got it. I'll revise error message.